### DISCUSSION TITLE
申请退款

### DISCUSSION THREAD

**Discussion by h4817517** [getlantern/lantern-forum-cn#466] [General]
这几个月体验太差了，咬牙又买了2年的，结果现在每次连接都非常困难，特别是电信网络
无奈申请退款

**Comment by MaxMKMax** [comment]
@h4817517 
您好, 请问您可以向我们的支付团队发送电子邮件吗？因为我们想保护您的个人信息和隐私。因此，最好直接向我们的支付团队发送电子邮件申请退款。谢谢。一般整个退款过程需要 14 天 - 20 天. 请您耐心等待. 谢谢您! 希望您过一阵子再回来用蓝灯, 说不定我们已经修复好所有的问题了. 真心希望您可以再次.成为我们的用户. 随时欢迎回来. 谢谢您这段时间支持蓝灯!

电子邮件：support@getlantern.org

**Comment by MaxMKMax** [comment]
@h4817517 您方便告诉我体验感差时遇到那些问题了吗? 也希望您帮我提交各应用内报告. 我来反馈, 我们也希望会做得更好. 提交后给我留言 8 位数票号. 谢谢您!

**Comment by h4817517** [comment]
我之前的描述不正确
现在的情况是显示连接正常，但没有
流量，需要多次关闭打开蓝灯才可以，并且还经常卡顿。
在 2024年4月25日星期四，Mike K ***@***.***> 写道：

**Reply by MaxMKMax** [reply]
@h4817517 好的. 明白的. 可以帮我提交各应用内报告吗? 然后通知我票号. 谢谢!

您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
@h4817517 退款的话, 您直接发付款单据和您的邮箱给我吧. 我帮您申请上去. 非常抱歉影响了您的体验感. 

希望您过几个月后再回来使用看蓝灯. . 我们现在正在大修复了. 估计不久后就会恢复稳定. 希望您以后再回来成为我们专业版的用户. 欢迎以后再回来. 谢谢您支持我们!

**Comment by h4817517** [comment]
***@***.***
当时是网上付款，银行回应一个短信可以吗？还是需要别的付款单据？

在 2024年5月2日星期四，Mike K ***@***.***> 写道：

**Reply by MaxMKMax** [reply]
@h4817517 您付款之后一般会收到单据的. 我们需要那个单据.

**Comment by MaxMKMax** [comment]
@h4817517 您的邮箱被GitHub屏蔽了. 请您隔开邮箱. 比如: s upport @ getlanter n. org
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/7fadce33-40a3-4cb7-b48f-2d10eac5297c)

**Comment by h4817517** [comment]
单据是发到我邮箱还是付款后直接显示？在哪里还能找到吗？

在 2024年5月8日星期三，Mike K ***@***.***> 写道：
